sentences/fr/homeassistant_HassTimerStatus.yaml

```diff
@@ -6,56 +6,62 @@ intents:
       # No name
       - sentences:
           # Combien de temps reste-t-il
-          - "Combien de temps <reste_t_il>"
+          - "[Encore] Combien de temps <reste_t_il>"
           # Combien de temps reste-t-il au minuteur
-          - "Combien de temps <reste_t_il> au <minuteur>"
+          - "[Encore] Combien de temps <reste_t_il> au <minuteur>"
           # Combien de temps reste-t-il sur le minuteur
-          - "Combien de temps <reste_t_il> (sur|dans) [(le|mon)] <minuteur>"
+          - "[Encore] Combien de temps <reste_t_il> (sur|dans) [(le|mon)] <minuteur>"
           # Il reste combien de temps
-          - "Il reste combien [de temps]"
+          - "[Il reste] [encore] combien [de temps]"
           # Il reste combien de temps au minuteur
-          - "Il reste combien [de temps] au <minuteur>"
+          - "[Il reste] [encore] combien [de temps] au <minuteur>"
           # Il reste combien de temps sur le minuteur
-          - "Il reste combien [de temps] (sur|dans) [(le|mon)] <minuteur>"
+          - "[Il reste] [encore] combien [de temps] (sur|dans) [(le|mon)] <minuteur>"
```

Comment:
@piitaya 
Shortest sentences is now "Combien sur mon minuteur?"
Minuteur is mandaotry.
I'm **OK** with that.
Agreed?